src/fetch_data.py

```diff
@@ -12,21 +12,31 @@ def main():
         "operationName": "getTransactions",
         "variables": None,
         "query": """query getTransactions {
-            nftTransferEntities(limit:30000,where: {toAddress_not_contains: "0x00", fromAddress_not_contains: "0x00"}) {
+            nftTransferEntities(
+                limit: 10000,
+                where: {
+                    toAddress_not_contains: "0x00",
+                    fromAddress_not_contains: "0x00",
+                    
+                }
+            )
+            {
                 amount
                 id
                 fromAddress
                 toAddress
                 nft {
-                id
+                    id
+                    name
+                    description
+                    uri
```

Comment:
Here you can also add attributes, its a map of features that NFT is made of, we can take those and also save them to DB. Later we will analyze them and create tags based on them

Reply:
✅